users/versioning.rst

```diff
@@ -170,21 +170,21 @@ behavior as mentioned above. I created the following script and saved it as
     chcp 65001
 
     rem We need command extensions for md to create intermediate folders in one go
-    setlocal EnableExtensions
+    setlocal enableextensions
```

Comment:
Oh nice, the best of all the cASev_AriantS: `novisualseparationnomatterhowmanywordswesticktogether` (only trumped by `SCREAMINGNOVISUALSEPARATION`).

Reply:
Yeah, although to be fair, these old commands are usually short enough not make any real difference regardless of the case. Note that environment variables still use UPPERCASE 😉.